resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1

```diff
@@ -584,6 +595,128 @@ class TextCursor
     }
 }
 
+[DSCResource()]
+class ColorFilter
+{
+    # Key required. Do not set.
+    [DscProperty(Key)] [string] $SID
+    [DscProperty()] [nullable[bool]] $FilterStatus
+    [DscProperty()] [ColorFilters] $FilterColor
+    [DscProperty()] [nullable[bool]] $KeyboardShortcutStatus
+
+    static hidden [string] $FilterStatusProperty = 'Configuration'
+    static hidden [string] $FilterStatusActiveProperty = 'Active'
+    static hidden [string] $FilterStatusValue = 'colorfiltering'
+    static hidden [string] $FilterColorProperty = 'FilterType'
+    static hidden [string] $KeyboardShortcutStatusProperty = 'HotkeyEnabled'
+
+    static [bool] GetStatus()
+    {
+        $FilterStatusArgs = @{  Path = $global:NTAccessibilityRegistryPath; Name = ([ColorFilter]::FilterStatusProperty)}
+        if (-not(DoesRegistryKeyPropertyExist @FilterStatusArgs))
+        {
+            return $false
+        }
+        else
+        {
+            $colorFilterSetting = (Get-ItemProperty @FilterStatusArgs).Configuration

```

Comment:
Is .Configuration the correct field? Is it not `Active`? What's the difference between configuration and active

Reply:
The commented notes that I had removed:

```PowerShell
<#
Enable/disable is controlled by 2 keys:
- HKEY_CURRENT_USER\Software\Microsoft\ColorFiltering
  - Dword: "Active"
- HKEY_CURRENT_USER\Software\Microsoft\Windows NT\CurrentVersion\Accessibility
  - Key: "Configuration" add value "colorfiltering"
#>
```